### DISCUSSION TITLE
currencies (USD)

### DISCUSSION THREAD

**Discussion by theshape** [atrocore/atrocore#289] [Q&A] [answered]
are currencies just broken for now, or has the ability to not use Euros been completely relegated to the optional Pricing module now?
/#Currency/view is just blank in my installs and the demo.

**Comment by atrolex** [comment]
Thank you, It was a bug, we just fixed it. Please update.

**Comment by rratsun** [comment, marked as answer]
The problem is solved in **core 1.4.19**.
